.github/workflows/integrate.yml

```diff
@@ -0,0 +1,19 @@
+name: Node Continuous Integration
+
+on:
+    push: 
+        branches: [ main ]
+    pull_request:
+        branches: [ main ]
+
+jobs: 
+    build: 
+        runs-on: ubuntu-latest
+        steps: 
+            - uses: actions/checkout@v2
+            - uses: actions/setup-node@v2
+              with:
+                node-version: 18
+            - run: npm install
```

Comment:
Can we change to yarn? Since we use yarn instead of npm

Reply:
Reference:
https://michaelcurrin.github.io/code-cookbook/recipes/ci-cd/github-actions/workflows/node/yarn.html
@nicolas-lukita